crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -718,7 +733,12 @@ impl UnableToDecryptReason {
     /// Returns true if this UTD is due to a missing room key (and hence might
     /// resolve itself if we wait a bit.)
     pub fn is_missing_room_key(&self) -> bool {
-        matches!(self, Self::MissingMegolmSession | Self::UnknownMegolmMessageIndex)
+        match self {
```

Comment:
The doc says here that `true` means that  `( [it] might resolve itself if we wait a bit.)`
If we have a withheld code waiting won't help. It is used by the notification client to decide if it should work for an additional encryption sync.